worker/include/Channel/ChannelRequest.hpp

```diff
@@ -60,13 +57,14 @@ namespace Channel
 
 	private:
 		void Send(uint8_t* buffer, size_t size);
-		void Send(const flatbuffers::Offset<FBS::Response::Response>& reponse);
+		void SendResponse(const flatbuffers::Offset<FBS::Response::Response>& response);
```

Comment:
Why Send() and SendResponse()? Perhaps we should have Notify and Reply() instead?

Reply:
From what I understand this one is calling `Send` internally, `Send` only operates on bytes, while this operates on strongly typed response. Jose should be able to tell more when he is back.

Reply:
Correct. Both methods are private. `Send(Response)`  sends exactly that and the other the given buffer. I'll remove `Send(buffer, size)` after this PR is merged since it's a one line method used in a single place.